CONTRIBUTING.md

```diff
@@ -166,10 +167,10 @@ documentation:
 
 #### Dashboard images (screenshots)
```

Comment:
We should probably update this heading so people know images and screenshots are different things.
```suggestion
#### Dashboard Screenshots
```
And then have a separate heading for images displayed on the dashboard